packages/rocketchat-slackbridge/package.js

```diff
@@ -15,8 +15,13 @@ Package.onUse(function(api) {
 
 	api.addFiles('logger.js', 'server');
 	api.addFiles('settings.js', 'server');
```

Comment:
Broke up the logic from slackbridge into rocket specific and slack specific files